Circles/Views/Composer/PostComposer.swift

```diff
@@ -11,8 +11,17 @@ import PhotosUI
 import QuickLookThumbnailing
 import Matrix
 
+class MessageStatus: ObservableObject {
```

Comment:
So, it's a bit weird to use an ObservableObject here, when we're only passing a single boolean from one View down to the next one in the hierarchy.

Feels like this is should be more like a Binding.

Reply:
I agree that this is not perfect, but with @Binding instead I need to use pass it in each place where we call PostComposer. As we can't set default value to false such as we can do it with ObservableObject.